MrRio/jsPDF
# [Bugfix] Fix reference-server PDF creation

As described in #2526, the reference-server was creating malformed PDFs. This PR fixes the issue.

To test:

- Delete or rename `spec/reference/jpg.pdf` (or any reference PDF)
- Run the reference server: `npm run test-training`
- Run the local test suite: `npm run test-local`
- Check the newly created `jpg.pdf`
    - Before the fix, it produced a 21 KB file that appeared blank when opened (and fails the testing)
    - After the fix, it produces a 16 KB file that displays the dice image correctly (and passes the testing)
- I have only tested this in Windows

Closes #2526
